ember/app/ui/profile/index/controller.js

```diff
@@ -20,16 +21,36 @@ const queryParams = new QueryParams({
 export default class ProfileIndexController extends Controller.extend(
   queryParams.Mixin
 ) {
+  @service store;
```

Comment:
This controller has a lot of duplicated code. Maybe worth abstracting the pagination functionality? You could take a look at ember-resources for this task.

Reply:
Will do at a later date